components/openindiana/ca-certificates/Makefile

```diff
@@ -20,20 +20,22 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= ca-certificates
 COMPONENT_VERSION_MAJOR=3
-COMPONENT_VERSION_MINOR=76
+COMPONENT_VERSION_MINOR=77
 COMPONENT_VERSION=	$(COMPONENT_VERSION_MAJOR).$(COMPONENT_VERSION_MINOR)
 COMPONENT_SUMMARY=	Common CA certificates
 COMPONENT_SRC=		nss-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH= \
-		       sha256:1b8e0310add364d2ade40620cde0f1c37f4f00a6999b2d3e7ea8dacda4aa1630
+		       sha256:825edf5a2fd35b788a23ff80face591f82919ae3ad2b8f77d424a450d618dedd
 COMPONENT_ARCHIVE_URL= \
 	https://ftp.mozilla.org/pub/security/nss/releases/NSS_$(COMPONENT_VERSION_MAJOR)_$(COMPONENT_VERSION_MINOR)_RTM/src/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL = https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS
 COMPONENT_FMRI=		crypto/ca-certificates
 COMPONENT_LICENSE=	MPLv2.0
 COMPONENT_LICENSE_FILE=	nss/COPYING
 
+HUMAN_VERSION="20220309"
+
```

Comment:
It's not sufficient to add HUMAN_VERSION to the Makefile. You also need to reference it from the manifest file:
set name=pkg.human-version value=$(HUMAN_VERSION)